docs/modules/ROOT/pages/object-storage/create.adoc

```diff
@@ -29,6 +29,28 @@ spec:
 <4> Secret where the connection details are provisioned. This secret shouldn't exist at this time.
 +
 IMPORTANT: Object Storage Bucket names must be unique across providers. Make sure to choose a name with prefixes and other guards, to prevent any issue during the creation of the bucket. No error message is returned if the bucket cannot be created.
+== Create a Bucket in another cloud provider
```

Comment:
```suggestion

== Create a Bucket in another Cloud Provider
```

Missing newline at the top. Please always do a preview locally before handing out docs for review. Thanks.